client-sdk/go/client/api_cluster_record_keeping.go

```diff
@@ -1,11 +1,11 @@
 /*
-Copyright 2022 VMware, Inc.
+Copyright 2024 VMware, Inc.
```

Comment:
nit: You need not change the copyright year for existing files. Only if we're introducing new files, we add the current year.  
Or was this change auto-generated?

Reply:
This was not auto-generated. I have reverted copyright year changes from existing files.